assets/js/blocks/featured-product/style.scss

```diff
@@ -13,11 +13,19 @@
 	flex-wrap: wrap;
 	align-content: center;
 
+	&.components-resizable-box__container {
+		overflow: visible;
+	}
+
 	&.components-placeholder {
 		// Reset the background for the placeholders.
 		background-color: rgba( 139, 139, 150, .1 );
 	}
 
+	.components-resizable-box__handle {
+		z-index: 10;
```

Comment:
I'm a bit surprised we don't use sass variables for z-index settings. 🤔

Reply:
We could add them, if you want– it didn't seem a priority at the start. Want to add an issue about it?